### PR TITLE
treewide: remove nand0p as maintainer

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

While looking at the sphinx package (#122189) I noticed it was heavily
undermaintained, which is when we noticed nand0p has been inactive for
roughly 18 months. It is therefore prudent to assume they will not be
maintaining their packages, modules and tests.

- Their last contribution to nixpkgs was in 2019/12
- On 2021/05/08 I wrote them an email to the address listed in the
  maintainer-list, which they didn't reply to.

E-Mail contents: 
https://gist.github.com/mweinelt/4c2ffbb8214e58a7630b3da68e8c45a0

Open questions:
- How long do we want to wait for them to reply?
- Should we remove the maintainer handle as well?

What was done:
```
find pkgs -iname "*.nix" -exec sed -i 's/nand0p //' {} \\;
find nixos -iname "*.nix" -exec sed -i 's/nand0p //' {} \\;
```

cc @dotlambda @vcunat @nand0p 

###### Things done

- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/graphics/flam3/default.nix [hunk 1]
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Cosmic recursive fractal flames";
     homepage = "https://flam3.com/";
-    maintainers = [ maintainers.nand0p ];
+    maintainers = [ maintainers.];
```
Comment (dotlambda) [inline pkgs/tools/graphics/flam3/default.nix:20]:
```suggestion
    maintainers = with maintainers; [ ];
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by dotlambda** [comment]
> * How long do we want to wait for them to reply?

Why can always revert, so I see no reason to wait too long.



> * Should we remove the maintainer handle as well?

It's only fair to @nand0p to do so, as his personal data shouldn't be lingering around for no reason.

**Comment by mweinelt** [comment]
Done

**Review by dotlambda — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@mweinelt please fix the merge conflict.

**Comment by mweinelt** [comment]
Done

**Pull request merged**